src/main/java/wolox/training/models/User.java

```diff
@@ -23,7 +23,7 @@
     private long id;
 
     @NotNull
-    private String username;
+    private String userName;
```

Comment:
username is consider just a word, so was good :smile:

Reply:
change the variable name in a previous commit